doc/BorderRouter.rst

```diff
@@ -1,182 +1,220 @@
-# Border Router Redesign
+**********************************************
+Border Router Performance Optimized Redesign
+**********************************************
 
-* Author: Justin Rohrer
-* Last Updated: 2023-04-12
-* Status: In Review
-* Discussion at: [#4334](https://github.com/scionproto/scion/issues/4334)
-* Implemented in: [](will be added once implemented)
+- Author: Justin Rohrer
+- Last Updated: 2023-04-28
+- Discussion at: `#4334 <https://github.com/scionproto/scion/issues/4334>`_
 
-## Motivation
+Abstract
+===========
 
 Right now, the performance of the border router is very much limited because a single goroutine per
 border router interface is responsible for reading, parsing, processing and forwarding the packets.
+
+Background
+===========
+
 Redesigning this pipeline is expected to lead to significant performance improvements.
```

Comment:
- Move first sentence at the end of this paragraph. ("Redesigning the current router pipeline is...")
- Start with "The current border router design..."